x/ccv/consumer/keeper/relay.go

```diff
@@ -213,6 +216,16 @@ func (k Keeper) SendPackets(ctx sdk.Context) {
 			k.Logger(ctx).Error("cannot send IBC packet; leaving packet data stored:", "type", p.Type.String(), "err", err.Error())
 			break
 		}
+		// If the packet that was just sent was a Slash packet, set the waiting on slash reply flag.
+		// This flag will be toggled false again when consumer hears back from provider. See OnAcknowledgementPacket below.
+		if p.Type == ccv.SlashPacket {
+			k.UpdateSlashRecordOnSend(ctx)
+			// Break so slash stays at head of queue
+			break
+		} else {
+			// Otherwise the vsc matured will be deleted
+			idxsForDeletion = append(idxsForDeletion, p.Idx)
```

Comment:
Seems the `idxsForDeletion = append(idxsForDeletion, p.Idx)` is repeated here and below. Could this lead to cases where we try to delete a packet twice? Would it make sense to combine them to only `append` once?

Reply:
Nice catch, this was either a mistake on my part or a faulty merge conflict resolution 

fixed in https://github.com/cosmos/interchain-security/commit/a972fd4cb2a0edb44651ba39bd1157ae050b6f8d